airflowHPC/executors/radical_local_executor.py

```diff
@@ -103,48 +104,45 @@ def state_cb(task, state):
 
         self._rp_tmgr.register_callback(state_cb)
 
+        # start a service endpoint which listens for task execution requests
+        # sent from the main airflow command line process
+        self._zmq_server = ru.zmq.Server()
+        self._zmq_server.register_request("rp_execute", self._rp_execute)
+        self._zmq_server.start()
+
+        self._rp_endpoint = self._zmq_server.addr()
```

Comment:
```suggestion
        self._rp_endpoint = self._zmq_server.addr
```
This is a property, so not callable.

Reply:
oops :-)